doc/lua_api.txt

```diff
@@ -3057,6 +3057,15 @@ This is basically a reference to a C++ `ServerActiveObject`
         * `"skybox"`: Uses 6 textures, `bgcolor` used
         * `"plain"`: Uses 0 textures, `bgcolor` used
 * `get_sky()`: returns bgcolor, type and a table with the textures
+* `set_clouds(settings)`: set cloud parameters
+	* `settings` is a table with the following optional fields:
+		* `density`: from `0` (no clouds) to `1` (full clouds) (default `0.4`)
+		* `color`: cloud color during daylight (default `#fff0f0`)
+		* `ambient`: light-independent cloud color component, "glow at night" effect (default `#000000`)
+		* `height`: cloud height in blocks (default per conf, usually `120`)
```

Comment:
Ok height is fine for the name then, maybe the description can mention this is y of cloud base? Then 'nodes' doesn't need mentioning.